elements/hax-body/lib/hax-plate-context.js

```diff
@@ -205,6 +207,23 @@ class HaxPlateContext extends I18NMixin(HaxContextBehaviors(LitElement)) {
     }
   }
 
+  __dblClickFire(event) {
+    if (event.target.id === "remove") {
+      this.dispatchEvent(
+        new CustomEvent("hax-context-item-selected", {
+          bubbles: true,
+          cancelable: true,
+          composed: true,
+          detail: {
+            target: event.target,
+            eventName: event.target.eventName,
```

Comment:
no reason this couldn't just be `hax-plate-delete` that way we never accidentally wire up the previous element to emit an event of `hax-plate-delete`